tools/utils.c

```diff
@@ -237,16 +237,16 @@ void generate_filename(const char *in_name, char *out_name, char *extension)
    sprintf(out_name, "%s.%s", tmp_name, extension);
 }
 
-char *basename(const char *name)
-{
-   const char *base = name;
-   while (*name) {
-      if (*name++ == '/') {
-         base = name;
-      }
-   }
-   return (char *)base;
-}
+// char *basename(const char *name)
```

Comment:
Is this intentional? I'm fine with it. But it's part of a utils.c just wanna check this won't mess anything up.

Reply:
Yeah, it seems like the `basename` definition here is conflicting with something in C++'s STL (I think?). It goes unused in all our tools so commenting it out shouldn't hurt anything.